### PR TITLE
Feat/hero partner list 480

### PR DESCRIPTION
**Pull request by Hornebom**
Ticket:
https://app.zenhub.com/workspaces/centrifuge-applications-5fa29e924059e5001625e2a3/issues/centrifuge/website/480

### CODE REVIEW

File: src/components/partner-list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  partners: PartnerProps[]
+}
+
+export type PartnerProps = {
+  logo: keyof typeof logos
+  alt: string
+}
+
+const logos = {
+  aave: aave,
```
Comment (onnovisser) [inline src/components/partner-list/index.tsx:25]: can use shorthand format
```
const logos = {
  aave,
  ...
}
```

File: src/hooks/use-is-overflow.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+
+export const useIsOverflow = (ref: React.RefObject<HTMLDivElement>) => {
+  const [isOverflow, setIsOverflow] = React.useState(false)
+
+  React.useLayoutEffect(() => {
```
Comment (onnovisser) [inline src/hooks/use-is-overflow.tsx:6]: can't use `useLayoutEffect` with SSR

File: src/hooks/use-is-overflow.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    const trigger = () => {
+      const hasOverflow = current!.scrollWidth > current!.clientWidth
+
+      setIsOverflow(hasOverflow)
+    }
+
+    if (current) {
+      if ('ResizeObserver' in window) {
+        new ResizeObserver(trigger).observe(current)
```
Comment (onnovisser) [inline src/hooks/use-is-overflow.tsx:17]: needs to unobserved when the hook dismounts

File: src/components/partner-list/index.tsx (hunk 1 above)
Comment (Hornebom) [reply src/components/partner-list/index.tsx]: refactored

File: src/hooks/use-is-overflow.tsx (hunk 2 above)
Comment (Hornebom) [reply src/hooks/use-is-overflow.tsx]: refactored to use `useEffect` instead

File: src/hooks/use-is-overflow.tsx (hunk 3 above)
Comment (Hornebom) [reply src/hooks/use-is-overflow.tsx]: Added return stement and `unobserve` option

**Review by onnovisser — APPROVED** [review]

**Pull request merged**